src/Config/Provider/AutodnsProvider.php

```diff
@@ -0,0 +1,1347 @@
+                    "adminc" => [
+                        "id" => $authData["contactid"],
+                    ],
+                    "techc" => [
+                        "id" => $authData["contactid"],
+                    ],
+                    "confirmOrder" => $authData["ownerConfirm"],
+                    "nameServers" => [
+                        [
+                            "name" => "a.ns14.net"
+                        ],
+                        [
+                            "name" => "b.ns14.net"
+                        ],
+                        [
+                            "name" => "c.ns14.net"
+                        ],
+                        [
+                            "name" => "d.ns14.net"
+                        ],
+                    ]
+                ])
+                ->toArray()
+        )->toArray();
+
+    }
+
+    /**
+     * @throws TransportExceptionInterface
+     */
+    public static function verifyAuthData(array $authData, HttpClientInterface $client): array
+    {
+
+        $username = $authData['username'];
+        $password = $authData['password'];
+
+        $acceptConditions = $authData['acceptConditions'];
+        $ownerLegalAge = $authData['ownerLegalAge'];
+        $waiveRetractationPeriod = $authData['waiveRetractationPeriod'];
+
+        if (empty($authData["context"])) {
+            $authData["context"] = 4;
+        }
+
+        if (
+            empty($username) || empty($password)
+        ) {
+            throw new BadRequestHttpException('Bad authData schema');
+        }
+
+        if (
+            true !== $acceptConditions
+            || empty($authData['ownerConfirm'])
+            || true !== $ownerLegalAge
+            || true !== $waiveRetractationPeriod
+        ) {
+            throw new HttpException(451, 'The user has not given explicit consent');
+        }
+
+        try {
+            $response = $client->request(
+                'GET',
+                '/v1/hello',
+                (new HttpOptions())
+                    ->setAuthBasic($authData["username"], $authData["password"])
+                    ->setHeader('Accept', 'application/json')
+                    ->setHeader('X-Domainrobot-Context', $authData["context"])
+                    ->setBaseUri(self::BASE_URL)
+                    ->toArray()
+            );
+        } catch (\Exception $exp) {
+            throw new BadRequestHttpException('Invalid Login');
+        }
+
+        if (Response::HTTP_OK !== $response->getStatusCode()) {
+            throw new BadRequestHttpException('The status of these credentials is not valid');
+        }
+
+        return $authData;
+    }
+
+    /**
+     * @throws TransportExceptionInterface
+     * @throws ServerExceptionInterface
+     * @throws RedirectionExceptionInterface
+     * @throws DecodingExceptionInterface
+     * @throws ClientExceptionInterface
+     */
+    protected function getSupportedTldList(): array
+    {
+        return [
```

Comment:
This should fetch purchasable suffixes using an API endpoint. If autodns doesn't provide such an endpoint, could you write a comment here explaining so and give the source for this list?

Reply:
I believe that maintaining a hardcoded list of supported TLDs within the project may not be ideal.

Instead, I suggest displaying a warning message to the user in such cases.
This would encourage them to check for themselves the compatibility of the domain names they want to track without having to constantly update the list.

Reply:
The Provider do not provide a API for supported TLD in default API. 
Their "Pricelist XML API" must be booked individual, which contains a list of supported TLDs.
So this provider cannot rely on availability.

I simple take that list from CSV Pricelist.

You are right. When I simple overwrite the "isSupported" function, always "return true" and add the mentioned warning, it is not ideal, too, but maybe a better solution.